sql/src/main/java/io/crate/metadata/Schemas.java

```diff
@@ -72,6 +74,7 @@ public Schemas(Settings settings,
         this.docSchemaInfoFactory = docSchemaInfoFactory;
         schemas.putAll(builtInSchemas);
         this.builtInSchemas = builtInSchemas;
+        this.defaultTemplateService = new DefaultTemplateService(clusterService);
```

Comment:
As discussed (writing it here ftr): Added this to Schemas because it doesn't add any additional external dependencies (clusterService & settings are already there) and it kind of fits with the responsibillity of Schemas (representation & management of meta data).

And I wanted to avoid having another guice usage. Putting it into Schemas will also make sure unit-tests which use the SqlExecutor use it as well (although it won't have an effect there yet, because I think we don't trigger clusterState changes there)